_posts/2020-02-18-bitcoin-abc-0.21.0.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Prepare for the Bitcoin Cash upgrade with Bitcoin ABC 0.21.0
+subtitle: May 15th 2020 protocol upgrade is locked in
+multiLangId: 2020-02-18-bitcoin-abc-0-21-0
+lang: en
+---
+
+Every six months, Bitcoin Cash takes one step closer to realizing its [roadmap](https://www.bitcoincash.org/roadmap.html) of global peer-to-peer electronic cash.  With this coming network upgrade, Bitcoin ABC has prepared its 0.21.0 release. This release is compatible with the network upgrade and can be downloaded at: [https://download.bitcoinabc.org/0.21.0/](https://download.bitcoinabc.org/0.21.0/)
+
+Similar to the last upgrade, the Bitcoin ABC 0.21.0 release has been made available three months in advance of the upgrade activation, which is scheduled to occur on May 15th, 2020. The release and a specification [1] are prepared well in advance of the upgrade to give all network participants ample time to update. The protocol upgrade, once activated, will enable the following features:
```

Comment:
Maybe include a link here? Like this:

Similar to the [last upgrade](https://www.bitcoinabc.org/2019-08-21-bitcoin-abc-0.20.0/)